app/stac_api/admin.py

```diff
@@ -271,6 +272,106 @@ def save_model(self, request, obj, form, change):
         return super().save_model(request, obj, form, change)
 
 
+@admin.register(CollectionAsset)
+class CollectionAssetAdmin(admin.ModelAdmin):
+
+    class Media:
+        js = ('js/admin/asset_help_search.js',)
+        css = {'all': ('style/hover.css',)}
+
+    autocomplete_fields = ['collection']
+    search_fields = ['name', 'collection__name']
+    readonly_fields = [
+        'collection_name',
+        'href',
+        'checksum_multihash',
+        'created',
+        'updated',
+        'etag',
+        'update_interval'
+    ]
+    list_display = ['name', 'collection_name', 'collection_published']
+    fieldsets = (
+        (None, {
+            'fields': ('name', 'collection', 'created', 'updated', 'etag')
+        }),
+        (
+            'File', {
+                'fields': ('file', 'media_type', 'href', 'checksum_multihash', 'update_interval')
```

Comment:
I'm not sure we need the `update_interval` here... would it be considered at all? If I remember correctly we had set a fixed cache duration for assets uploaded via admin...

Reply:
From discussion: cachecontrol header is set to a fixed value for assets uploaded via admin ui, so `update_interval` has no effect. We decided to keep it mainly to keep the two asset models and logic similar